platform/jobs/edxPlatformTestSubset.groovy

```diff
@@ -64,10 +65,9 @@ def secretMap = [:]
 try {
     out.println('Parsing secret YAML file')
     /* Parse k:v pairs from the secret file referenced by secretFileVariable */
-    Thread thread = Thread.currentThread()
-    Build build = thread?.executable
-    Map envVarsMap = build.parent.builds[0].properties.get("envVars")
-    secretMap = JENKINS_PUBLIC_PARSE_SECRET.call(secretFileVariable, envVarsMap, out)
+    String contents = new File("${EDX_PLATFORM_TEST_SUBSET_SECRET}").text
+    Yaml yaml = new Yaml()
+    secretMap = yaml.load(contents)
     out.println('Successfully parsed secret YAML file')
```

Comment:
This will need to be changed in all of the jobs, in order to work with the tests. Here is why:
When you run the dsl scripts via a Jenkins seed job, you have access to the full Jenkins run time, and you can manipulate Executors/Build data. However, when you run the spock tests, we are using a minimal version of Jenkins (not the same as a full fledged installation), so we can't access, say, build history or executors. I have tested this on a sandbox installation, and the ${} notation works for secrets (it is the normal way of accessing env vars within the Build steps of a job.